### DISCUSSION TITLE
Randomly playing a youtube video (sound only of course) from a specific channel.

### DISCUSSION THREAD

**Discussion by stucker9** [TannerGabriel/discord-bot#55] [Q&A]
-removed by author-

**Comment by TannerGabriel** [comment]
I have never attempted or thought about that myself before. That said, here are some ideas I came up with when thinking about it:

- You will need some kind of system to fetch the video URLs from the channel you want to play. This will probably be possible using the Youtube API or some other kind of library.
- After you can fetch the URLs, you can reuse the current play command code and play one song after the other as if you input multiple URLs into the current bot.

Hopefully, this helps you get started. If you still have questions, feel free to continue the conversation. I also wish you merry Christmas and a happy new year.

Best regards,
Gabriel
